webdriver/tests/bidi/network/add_intercept/invalid.py

```diff
@@ -1,3 +1,5 @@
+# META: timeout=long
+
```

Comment:
I'd expect invalid arguments tests to fail fast. Are you sure it is required?

Reply:
Removed for now.